plugins/syntax/g_numeral.mlg

```diff
@@ -40,12 +40,7 @@ END
 
 VERNAC COMMAND EXTEND NumeralNotation CLASSIFIED AS SIDEFF
   | #[ locality = Attributes.locality; ] [ "Number" "Notation" reference(ty) reference(f) reference(g) ":"
-      ident(sc) numnotoption(o) ] ->
+      ident(sc) numeral_modifier(o) ] ->
 
     { vernac_numeral_notation (Locality.make_module_locality locality) ty f g (Id.to_string sc) o }
-  | #[ locality = Attributes.locality; ] [ "Numeral" "Notation" reference(ty) reference(f) reference(g) ":"
-      ident(sc) numnotoption(o) ] ->
-
-    { warn_deprecated_numeral_notation ();
-      vernac_numeral_notation (Locality.make_module_locality locality) ty f g (Id.to_string sc) o }
```

Comment:
Wait a minute. This PR removes the deprecated `Numeral Notation` command but this command was only deprecated in 8.13, which is not released yet.

Reply:
OK, wainting.

By the way, won't it be too much of a burden for users to change from `Numeral` to `Number`? (Or maybe the changes are already clearly accepted upstreams?)

Reply:
> By the way, won't it be too much of a burden for users to change from `Numeral` to `Number`? (Or maybe the changes are already clearly accepted upstreams?)

That's a fair question. There has been a bit of a debate after the PR changing this was merged. See https://github.com/coq/coq/pull/12979#issuecomment-692278941. IMHO for this kind of cosmetic change, the solution is to make the deprecation period sufficiently long (a bit like the `Ocaml` vs `OCaml` value for `Extraction Language` which was changed and deprecated in 8.8 but only removed in 8.13).

Reply:
Yes, this is somehow the question. Is the added value of the change worth the burden (but again, if the change is already validated by the users, it does not matter)?

Reply:
My mistake during rebasing, now fixed.